interfaces/apparmor/template.go

```diff
@@ -488,6 +488,10 @@ var defaultTemplate = `
   # Allow read-access to / for navigating to other parts of the filesystem.
   / r,
 
+  # Snap-specific run directory (see 'parallel installs', above)
```

Comment:
This looks good. I should've mention this in the bug, but please adjust this to be:
```
# Snap-specific run directory. bind mount *not* used here
# (see 'parallel installs', above)
```

Reply:
Fixed and force pushed.